Translations/pt_BR.po

```diff
@@ -4861,17 +4817,15 @@ msgstr "Covil do Arcebispo Lázarus"
 
 #: Source/levels/setmaps.cpp:32
 msgid "Church Arena"
-msgstr ""
+msgstr "Arena da Masmorra"
```

Comment:
Shouldn't this be "Arena da Igreja"?

Reply:
"Dungeon" is being translated as _Masmorra_ in other places I am just keeping it consistent. I can change other places to Igreja, I have no strong opinion about it.

Reply:
Hmm.. I was basing my comment on "Church" Arena, not "Dungeon". Is this one of those cases where we should reconsider the English original as well then?

Reply:
I think the original could be changed, but some people my not agree. If I got it right, dungeon here means the first levels and some people might argue that even Hell is a dungeon. 🤷‍♂️

Reply:
But where is "Dungeon" coming from in this case? The original text to be translated is "Church Arena", correct?

I feel like I might be missing something on this one.